packages/frontend/src/pages/SavedQueries.tsx

```diff
@@ -1,21 +1,18 @@
-import { Button } from '@blueprintjs/core';
 import { subject } from '@casl/ability';
 import { LightdashMode } from '@lightdash/common';
-import { Stack } from '@mantine/core';
-import { IconChartBar } from '@tabler/icons-react';
+import { Button, Center, Group, Stack } from '@mantine/core';
+import { IconChartBar, IconPlus } from '@tabler/icons-react';
 import { FC } from 'react';
 import { Helmet } from 'react-helmet';
 import { useHistory, useParams } from 'react-router-dom';
-import Page from '../components/common/Page/Page';
-import { PageHeader } from '../components/common/Page/Page.styles';
```

Comment:
let's not forget to get rid of styles once we stop importing from here

Reply:
still being used elsewhere!